server/src/domain/person/person.dto.ts

```diff
@@ -12,6 +12,15 @@ export class PersonUpdateDto {
   @IsString()
   name?: string;
 
+  /**
+   * Person date of birth.
+   */
+  @IsOptional()
+  @IsDate()
+  @Transform(toDate)
```

Comment:
Can you replace this with `@Type(() => Date)`

Reply:
I couldn't handle null values if I used `@Type(() => Date`... I'll try another approach, also see my comment about API best practices below.

Reply:
You can add `ValidateIf((value) => value !== null)` instead.